RevenueCatUI/CustomerCenter/ViewModels/FeedbackSurveyViewModel.swift

```diff
@@ -57,10 +57,13 @@ class FeedbackSurveyViewModel: ObservableObject {
 
     func handleAction(
         for option: CustomerCenterConfigData.HelpPath.FeedbackSurvey.Option,
+        darkMode: Bool,
+        displayMode: CustomerCenterPresentationMode,
         dismissView: () -> Void
     ) async {
         if let customerCenterActionHandler = self.customerCenterActionHandler {
             customerCenterActionHandler(.feedbackSurveyCompleted(option.id))
+            trackSurveyOptionChosen(option: option, darkMode: darkMode, displayMode: displayMode)
```

Comment:
Hmm I'm wondering if we should do this before calling the handler so we make sure the event isn't lost in case of some error on the client handling code?